src/pages/messages/components/NviApprovals.tsx

```diff
@@ -13,35 +13,50 @@ interface NviApprovalsProps {
 
 export const NviApprovals = ({ approvals, totalPoints }: NviApprovalsProps) => {
   const { t } = useTranslation();
+  const checkedApprovals = approvals.filter(
+    (approval) => approval.status === 'Approved' || approval.status === 'Rejected'
+  );
 
   return (
-    <Box sx={{ m: '1rem' }}>
+    <Box sx={{ m: '1rem', border: '0.5px solid' }}>
```

Comment:
I skissene ser det ut som det skal være rosa border

```suggestion
    <Box sx={{ m: '1rem', border: '0.5px solid', borderColor: 'nvi.main' }}>
```